core/src/main/java/software/amazon/awssdk/core/http/async/UnmarshallingAsyncResponseHandler.java

```diff
@@ -1,71 +0,0 @@
-/*
- * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
this class is not being used and is a duplicate of https://github.com/aws/aws-sdk-java-v2/blob/master/core/src/main/java/software/amazon/awssdk/core/client/AsyncClientHandlerImpl.java#L281